### ISSUE TITLE
关于验证集的valid_step

### ISSUE THREAD

**Issue by JHXion9** [sonwe1e/Template#2]
为什么在valid_step中
不用加with torch.no_grad()

**Comment by sonwe1e** [comment]
你好，在 pl.LightningModule 中会直接将梯度取消，并且设置成验证模式
"""
Note:
    When the :meth:`validation_step` is called, the model has been put in eval mode
    and PyTorch gradients have been disabled. At the end of validation,
    the model goes back to training mode and gradients are enabled.

"""

**Comment by JHXion9** [comment]
THX，you make my day
